src/WinGet.RestSource.Utils/Models/Objects/InstallationMetadataFile.cs

```diff
@@ -0,0 +1,130 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="InstallationMetadataFile.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.WinGet.RestSource.Utils.Models.Objects
+{
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using Microsoft.WinGet.RestSource.Utils.Models.Core;
+    using Microsoft.WinGet.RestSource.Utils.Validators.EnumValidators;
+    using Microsoft.WinGet.RestSource.Utils.Validators.StringValidators;
+
+    /// <summary>
+    /// InstallationMetadataFile.
+    /// </summary>
+    public class InstallationMetadataFile : IApiObject<InstallationMetadataFile>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InstallationMetadataFile"/> class.
+        /// </summary>
+        public InstallationMetadataFile()
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets RelativeFilePath.
+        /// </summary>
+        [InstallationMetadataFileRelativeFilePathValidator]
+        public string RelativeFilePath { get; set; }
+
+        /// <summary>
+        /// Gets or sets DocumentUrl.
+        /// </summary>
+        [SignatureSha256Validator]
```

Comment:
>SignatureSha256Validator [](http://example.com/codeflow?start=9&length=24)

Just realized there's an InstallerSha256Validator and SignatureSha256Validator

We should probably merge the 2 validators since sha256 format should always be the same. 

If there's a reason for making them separate validators, this would be FileSha256Validator

Reply:
They are identical - but were separate when we made it. I'll fix this.